### ISSUE TITLE
Wand disappears when dropping it ( pressing Q )

### ISSUE THREAD

**Issue by TheKommandr** [elBukkit/MagicPlugin#1046]
I was recently playing Minecraft 1.16.5 with my friends, and I downloaded this Magic plugin for fun and it worked perfectly fine. I tried every feature of the plugin and found no issues. One day my friends joined the server and I showcased them the new magic plugin and they loved it. I told everyone to press Q to open the spell inventory, but when they opened it the wand disappeared when they tried to close it. I don't know if its a bug, or we're doing something wrong. I hope someone can help me fix this. Thanks.

**Comment by NathanWolf** [comment]
Does it happen every time?
Sounds like a bug if so, but not one anyone else is seeing. Could be due to some other plugin, do you have any plugins that would be doing something special when a player drops an item?

You can change the controls for your wand in the enchant table menu, try using right-click instead of drop if you can't figure out what is causing this.

**Comment by TheKommandr** [comment]
Is it possibly AuthMe plugin is causing this? Or maybe its because I have a lot of plugins active. I do have a lot of plugins and I'll try to remove them to see if it works.

**Comment by NathanWolf** [comment]
I doubt it's Authme, maybe ViaVersion if you have that but I also kind of doubt it.

More likely something that handles item drops, no idea what that might be.

If right-click works you can also change the defaults to that and maybe avoid the problem.

**Comment by TheKommandr** [comment]
Is there any way for me to change the default button for everyone? For example G or a specific button?

**Comment by NathanWolf** [comment]
No, the server doesn't read keyboard inputs like that from the game client. 

You are limited to using Minecraft controls, left click (break), right click (interact), drop and swap keys.
